DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -43,7 +43,7 @@
 from DIRAC.Resources.Storage.StorageBase import StorageBase
 
 
-BASE_PATH = ""
+BASE_PATH = "/"
```

Comment:
Is this change necessary?

https://github.com/DIRACGrid/DIRAC/blob/e0fc1aa48f785babb845f5a364e82b033eb3c60c/DataManagementSystem/Service/StorageElementHandler.py#L118-L130

With `BASE_PATH="/"` this would no longer fail, while with the empty string it will.

Reply:
It was just for the test. 
What if 
```
 BASE_PATH = getServiceOption(serviceInfo, "BasePath", BASE_PATH) 
```
is replaced by 
```
 BASE_PATH = getServiceOption(serviceInfo, "BasePath") 
```

Reply:
This PR starts to touch really a lot of things :-D Do you want me to check what the actual problem of LogSE is ? :-D

Reply:
Yes changing the getServiceCall restores previous behavior.
Maybe add a comment to the BASE_PATH that it is not a constant, but a configurable option. It is not nice having this "default" value ("/") not actually being used.